packages/@aws-cdk/aws-rds/lib/proxy.ts

```diff
@@ -341,6 +348,11 @@ abstract class DatabaseProxyBase extends cdk.Resource implements IDatabaseProxy
   public abstract readonly dbProxyArn: string;
   public abstract readonly endpoint: string;
 
+  /**
+   * Collection of all the proxy endpoints of this proxy
+   */
+  public readonly endpoints: string[] = [];
```

Comment:
I'm not sure if this adds any value. If you have added a bunch of endpoints you would probably want to know which _specific_ one you want to connect to.

Reply:
True